torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -411,6 +413,7 @@ void RequestCallbackImpl::processPythonRRefFetchCall(
       Message m =
```

Comment:
this `processPythonRemoteCall` function should take the ctx as an argument and insert streams to that ctx accordingly for each device used by the RRef.